blackbox/docs/config/logging.rst

```diff
@@ -40,114 +63,132 @@ configuration.
 And here is a snippet of the generated properties ready for use with log4j.
 You get the point.
 
-.. code-block:: yaml
+.. SEEALSO::
 
-  log4j.rootLogger=INFO, console
+    Consult the `PropertyConfigurator`_ documentation or the `configuration
+    section`_ of the Log4j documentation for more information.
 
-  log4j.logger.action=DEBUG
+.. _conf-logging-log4j-loggers:
 
-  log4j.appender.console=org.elasticsearch.common.logging.log4j.ConsoleAppender
-  log4j.appender.console.layout=org.apache.log4j.PatternLayout
-  log4j.appender.console.layout.conversionPattern=[%d{ISO8601}][%-5p][%-25c] %m%n
+Loggers
+.......
 
-   ...
+.. FIXME: need a full list of available loggers
 
-.. _PropertyConfigurator: https://logging.apache.org/log4j/1.2/apidocs/org/apache/log4j/PropertyConfigurator.html
+Possible log levels are the same as for Log4j_: ``TRACE``, ``DEBUG``, ``INFO``,
+``WARN``, and ``ERROR``. They must be provided as string literals in the
+``SET`` statement.
+
+.. NOTE::
+
+   Be careful using the ``TRACE`` log level because it's extremely verbose,
+   can obscure other important log messages and even fill up entire data disks
+   in some cases.
 
-Settings
---------
+.. FIXME: how to configure root logger?
 
-It's possible to set the log level of loggers at runtime. This is particularly
-useful when debugging problems and there is a need to increase the log level
-without wanting to restart nodes. Logging settings are cluster wide and
-override the logging configuration of nodes defined in their
-``log4j2.properties``.
+.. _conf-logging-log4j-run-time:
 
-The :ref:`RESET<ref-set>` statement is also supported, however only with the
-limitation that the reset of the logging override only takes affect after
-cluster restart.
+Run-Time Configuration
+......................
 
-To set the log level you can use the regular :ref:`SET<ref-set>` statement, for
-example:
+It's possible to set the log level of loggers at runtime using :ref:`SET
+<ref-set>`, like so:
 
 .. code-block:: sql
 
   SET GLOBAL TRANSIENT "logger.action" = 'INFO';
 
-The logging setting consists of the prefix ``logger`` and a variable suffix
-which defines the name of the logger that the log level should be applied to.
+In this example, the log level ``INFO`` is applied to the ``action`` logger.
 
-In addition to `hierarchical named loggers`_ you can also change the log level
-of the root logger using the ``_root`` suffix.
+In addition to being able to configure any of the standard :ref:`loggers
+<conf-logging-log4j-loggers>`, you can configure the root (i.e. default) logger
+using ``logger._root``.
 
-In the example above the log level ``INFO`` is applied to the logger
-``action``.
+.. FIXME: above correct?
```

Comment:
@mxm see here also

specifically, is ``logger._root`` correct here?

Reply:
Looks correct.